geospaas/base_viewer/templates/base_viewer/origin_of_templates.html

```diff
@@ -0,0 +1,33 @@
+{% load static %}
```

Comment:
About the names of the templates files:
  - you probably don't need to include the word "template" in the names, since they are in the `templates` directory
  - in would be nice to have names that describe what the templates contain, for example `page_structure.html` and `elements.html`

Reply:
done